assets/pages/contributions-landing/contributionsLandingUS.jsx

```diff
@@ -4,42 +4,28 @@
 
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { createStore, applyMiddleware } from 'redux';
+import { applyMiddleware } from 'redux';
 import thunkMiddleware from 'redux-thunk';
 import { Provider } from 'react-redux';
 
 import SimpleHeader from 'components/headers/simpleHeader/simpleHeader';
 import SimpleFooter from 'components/footers/simpleFooter/simpleFooter';
 import ContribLegal from 'components/legal/contribLegal/contribLegal';
 
-import pageStartup from 'helpers/pageStartup';
-import { getQueryParameter } from 'helpers/url';
-import { setCountry } from 'helpers/internationalisation/country';
+import { init as pageInit } from 'helpers/page/page';
 
 import reducer from './reducers/reducers';
 import { saveContext } from './helpers/context';
 import ContributionsBundleContent from './components/contributionsBundleContent';
 
 
-// ----- Page Startup ----- //
-
-const participation = pageStartup.start();
-const country = 'US';
-setCountry(country);
-
-
 // ----- Redux Store ----- //
-
-const store = createStore(reducer, {
-  intCmp: getQueryParameter('INTCMP'),
-  isoCountry: country,
-  refpvid: getQueryParameter('REFPVID'),
+const store = pageInit(reducer, {
+  common: { isoCountry: 'US' },
```

Comment:
Unfortunately I think this will completely overwrite the common state with *just this field*, due to the way that [preloaded state](http://redux.js.org/docs/recipes/reducers/InitializingState.html) works in Redux. It's an irritating problem that I couldn't think of a solution for in #215; it's a pain to have to specify the entire common state every time you want to change just one field.

Looking at it now, I think there are a couple of better options:

1) Your suggestion of flattening the state, doing away with siloed `common` and `page` states. In this situation I think `preloadedState` should do a merge.
2) Not using preloaded state, and in `page.js` passing in an overwrite object and doing `Object.assign` with the initial state, which should merge the fields. I'm happy to make this change in a separate PR.

All of that said, in **this case** none of this should be necessary, as the `detect` function should parse 'US' from the path on this page, which means you don't have to pass it in manually 🙂